### PR TITLE
Fixes links in RSS feed

### PR DESCRIPTION
**Pull request by dancork**
## What does this PR do?
URL was missing in config so RSS links didn't work

## How to test
- Checkout branch
- Run `jekyll serve`
- Visit `http://localhost:4000/feed.xml`
- All links should start `http://holidayextras.github.com`

- Run `jekyll serve --config _config.yml,_config_dev.yml`
- Visit `http://localhost:4000/feed.xml`
- All links should start `http://localhost:4000`

#### Quality checklist (for PR author to complete BEFORE code review):
- [x] I've updated documentation with any changes to procedures..
- [x] I am ready for this to be code reviewed, merged and tested.

#### IP or Developer Review:
- [x] I’ve witnessed the work behaving as expected.
- [x] I’ve checked for coding anti-patterns.
- [x] +1 from me!

#### Software Engineer, Software Architect or Developer review:
- [x] I’ve witnessed the work behaving as expected.
- [x] I’ve checked for coding anti-patterns.
- [x] +1 from me!

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -4,7 +4,8 @@ markdown: kramdown
 kramdown:
   input: GFM
 permalink: pretty
-baseurl: /
+url: http://holidayextras.github.com
+baseurl:
```
Comment (jodiedoubleday) [inline _config.yml]: Removing the `/` in baseurl has resulted in all navigation and image links being broken.

**Comment by dancork** [comment]
@jodiedoubleday fixed. updated all links and created a local "dev" config. updated the readme.

File: _config.yml (hunk 1 above)
Comment (dancork) [inline _config.yml]: This is the correct setup, see answer 1...

http://stackoverflow.com/questions/27386169/change-site-url-to-localhost-during-jekyll-local-development

**Comment by storer** [comment]
The xml feed is broken due to a parsing error on an html entity - I think we need to run the post.url through an additional filter such as:
post.url | replace: '&', '&amp;'

**Comment by dancork** [comment]
@storer we shouldn't have ampersands in post urls. i'm more inclined to change the url.

**Comment by storer** [comment]
:+1: Makes sense since the title can be overwritten in the markdown

**Comment by dancork** [comment]
Cool, fixing it now. Adding redirect for the old url.

**Comment by dancork** [comment]
@jodiedoubleday @storer made changes, could you look again

**Comment by storer** [comment]
:+1:

**Comment by jodiedoubleday** [comment]
Look good, great changes :) :+1:

**Pull request merged**